src/codeu/chat/client/ClientUser.java

```diff
@@ -59,23 +59,23 @@ static public boolean isValidName(String userName) {
     return clean;
   }
 */
-
+    
 // Validate the username string
 public boolean isValidName(String userName) {
+	boolean clean = true; 
```

Comment:
just a small nit: you have some extra white space on this line. As per the Google style guide, you would want the indentation to be uniform between line in the same block.